docs/getting-started/installing-with-docker.md

```diff
@@ -37,13 +37,15 @@ external_id = [...]
 credential_source = Ec2InstanceMetadata
 ```
 
-Note that the [role ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) must also be provided to the [worker](../concepts/components/worker.md) as `RESOTOWORKER_AWS_ROLE`. This role is assumed while fetching the list of resources in each sub-account when `RESOTOWORKER_AWS_SCRAPE_ORG=true`.
+Note that the [role ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) must also be provided to the [worker](../concepts/components/worker.md) as `aws.role`. This role is assumed while fetching the list of resources in each sub-account when `aws.scrape_org=true`.
```

Comment:
```suggestion
Note that the [role ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) must also be provided to the [worker](../concepts/components/worker.md) as `aws.role`. This role is assumed while fetching the list of resources in each sub-account when `aws.scrape_org` is `true`.
```
It currently says `RESOTOWORKER_AWS_SCRAPE_ORG=true` only because that's how the env var is defined.